tests/integration_tests/_utils/start_tidb_cluster_impl

```diff
@@ -17,7 +17,7 @@ random_file_name=
 randomGenSocketsConf() {
 	random_str=$(date '+%s%N')
 	if [ "$(uname)" == "Darwin" ]; then
-		random_str=$(cat /dev/random | LC_CTYPE=C tr -dc "a-zA-Z0-9" | head -c 10)
+		random_str=$(cat /dev/random | LC_ALL=C tr -dc "a-zA-Z0-9" | head -c 10)
```

Comment:
Why we need to change this?

Reply:
to make  sure `tr` command can be executed.
```
cat /dev/random | LC_CTYPE=C tr -dc "a-zA-Z0-9" | head -c 10
tr: Illegal byte sequence

```